src/domain/event/__tests__/EditEventPage.test.tsx

```diff
@@ -26,52 +36,121 @@ beforeEach(() => {
   });
 });
 
+const placeId = 'tprek:15417';
+
 const keywordMockResponse = {
-  keyword: {
+  keyword: fakeKeyword({
     id: 'yso:p4363',
-    name: {
-      en: 'families',
-      fi: 'perheet',
-      sv: 'familjer',
-      __typename: 'LocalisedObject',
-    },
-    internalId: 'https://api.hel.fi/linkedevents-test/v1/keyword/yso:p4363/',
-    __typename: 'Keyword',
-  },
+    name: fakeLocalizedObject('perheet'),
+  }),
 };
 
 const venueQueryResponse = {
   data: {
-    venue: {
-      id: 'tprek:15417',
+    venue: fakeVenue({
+      id: placeId,
       hasClothingStorage: true,
       hasSnackEatingPlace: true,
       translations: [
         {
-          languageCode: 'FI',
+          languageCode: 'FI' as any,
           description: 'Testitapahtuman kuvaus',
           __typename: 'VenueTranslationType',
         },
       ],
-      __typename: 'VenueNode',
-    },
+    }),
   },
 };
 
-const venueQueryResponseMock = {
+const keywordResponse = {
   data: {
-    venue: {
-      id: 'tprek:15417',
-      hasClothingStorage: true,
-      hasSnackEatingPlace: true,
-      translations: [
+    keyword: fakeKeyword({
+      id: 'yso:p4363',
+      name: fakeLocalizedObject('perheet'),
+    }),
+  },
+};
+
+const placeResponse = {
+  data: {
+    place: fakePlace({ name: fakeLocalizedObject('Sellon kirjasto') }),
+  },
+};
+
+const profileResponse = {
+  data: {
+    myProfile: fakePerson({
+      organisations: fakeOrganisations(1, [
         {
-          languageCode: 'FI',
-          description: 'Testitapahtuman kuvaus',
-          __typename: 'VenueTranslationType',
+          id: 'T3JnYW5pc2F0aW9uTm9kZTox',
+          persons: fakePersons(1, [
+            {
+              organisations: [] as any,
+              name: 'Testaaja2',
+              id:
+                'UGVyc29uTm9kZTo0MGZmYTIwMS1mOWJhLTQyZTYtYjY3Ny01MWQyM2Q4OGQ4ZDk=',
+            },
+          ]),
+          name: 'Kulttuurin ja vapaa-ajan toimiala',
         },
+      ]),
+    }),
+  },
+};
+
+const eventResponse = {
```

Comment:
You could have constant for all of these event fields. You have typed same texts also to tests

Reply:
added variables